test/libsolidity/semanticTests/viaYul/conditional/conditional_multiple.sol

```diff
@@ -0,0 +1,10 @@
+contract A {
+	function f() public pure returns (uint) {
+		uint x = 3 < 0 ? 2 > 1 ? 2 : 1 : 7 > 2 ? 7 : 6;
+		return x;
+	}
+}
+// ====
+// compileViaYul: true
```

Comment:
Can you set the tests to ``compileViaYul: also``? Otherwise I need to actually check, if the results are right :-D...